docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -85,15 +85,18 @@ The connector supports pushdown for a number of operations:
 * :ref:`limit-pushdown`
 * :ref:`topn-pushdown`
 
+.. _singlestore-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
+The connector provides read access and write access to data and metadata in
```

Comment:
This connector (you are not disambiguating the connector from another component, but from other connectors)